products/bridge/smart-contracts/contracts/core/ChainDispatcherWithoutFees.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.20;
+
+import {MessageHashUtils} from "@openzeppelin/contracts/utils/cryptography/MessageHashUtils.sol";
+import {ValidatorManager} from "contracts/core/ValidatorManager.sol";
+import {FeeTracker} from "contracts/core/FeeTracker.sol";
+import {DispatchReplayChecker} from "contracts/core/DispatchReplayChecker.sol";
+
+interface IChainDispatcherEvents {
+    event Dispatched(
+        uint indexed sourceChainId,
+        address indexed target,
+        bool success,
+        bytes response,
+        uint indexed nonce
+    );
+}
+
+interface IChainDispatcherErrors {
+    error NonContractCaller();
+    error NotValidator();
+}
+
+interface IChainDispatcher is IChainDispatcherEvents, IChainDispatcherErrors {}
+
+// Cross-chain only
+contract ChainDispatcherWithoutFees is IChainDispatcher, DispatchReplayChecker {
```

Comment:
Upon deployment, I'll be using this version of the dispatcher that does not include the whole fee management feature. Me and @DrZoltanFazekas agreed to skip this on this version.
However the tests are still based on the version with the fee management feature